go.mod

```diff
@@ -20,7 +21,7 @@ require (
 	sigs.k8s.io/controller-runtime v0.14.3
 	sigs.k8s.io/controller-tools v0.11.3
 	sigs.k8s.io/kustomize v2.0.3+incompatible
-	vitess.io/vitess v0.10.3-0.20230225051837-12cd2f303f5f
+	vitess.io/vitess v0.10.3-0.20230406205436-137cf9daf411
```

Comment:
that's a massive gap, we've been missing quite a few dependency upgrade. Hopefully it does not break a lot.